pylot/perception/detection/detection_operator.py

```diff
@@ -38,42 +39,14 @@ def __init__(self, camera_stream, time_to_decision_stream,
         self._flags = flags
         self._logger = erdos.utils.setup_logging(self.config.name,
                                                  self.config.log_file_name)
-        self._detection_graph = tf.Graph()
-        # Load the model from the model file.
-        pylot.utils.set_tf_loglevel(logging.ERROR)
-        with self._detection_graph.as_default():
-            od_graph_def = tf.GraphDef()
-            with tf.gfile.GFile(model_path, 'rb') as fid:
-                serialized_graph = fid.read()
-                od_graph_def.ParseFromString(serialized_graph)
-                tf.import_graph_def(od_graph_def, name='')
-
-        self._gpu_options = tf.GPUOptions(
-            allow_growth=True,
-            visible_device_list=str(self._flags.obstacle_detection_gpu_index),
-            per_process_gpu_memory_fraction=flags.
-            obstacle_detection_gpu_memory_fraction)
```

Comment:
We still need to support GPUOptions for our models. This issue has a discussion on how to do that in TFv2.0:
https://github.com/tensorflow/tensorflow/issues/25138